js/src/name-pubsub/cancel.js

```diff
@@ -0,0 +1,80 @@
+/* eslint max-nested-callbacks: ["error", 5] */
+/* eslint-env mocha */
+'use strict'
+
+const loadFixture = require('aegir/fixtures')
+
+const { spawnNodeWithId } = require('../utils/spawn')
+const { getDescribe, getIt, expect } = require('../utils/mocha')
+
+const fixture = Object.freeze({
+  data: loadFixture('js/test/fixtures/testfile.txt', 'interface-ipfs-core'),
+  cid: 'Qma4hjFTnCasJ8PVp3mZbZK5g2vGDT4LByLJ7m8ciyRFZP'
```

Comment:
IMHO we should just use the CID in the result from add and not hard code it. We're not testing "add" functionality here and if the CID changes in the future (because of changes in default dag layout or cid version for instance) there's fewer things to update in the tests. If you do change this then please also change in the other test files :)

Reply:
Will change in both